glusterd2/bricksplanner/planner.go

```diff
@@ -116,6 +116,9 @@ func getBricksLayout(req *api.VolCreateReq) ([]api.SubvolReq, error) {
 	subvolSize := req.Size
 	if numSubvols > 1 {
 		subvolSize = subvolSize / uint64(numSubvols)
+		if subvolSize < minBrickSize {
```

Comment:
Brick size in case of disperse Volume will be less than subvolsize.(that is `subvolsize / disperse_data_count`)

Reply:
this situation is taken care of.